lib/l10n/intl_es.arb

```
@@ -411,5 +411,10 @@
   "spanish": "Español",
   "italian": "Italiano",
   "chooseLanguageDescription": "Elige tu idioma preferido o usa el predeterminado del sistema",
-  "done": "HECHO"
+  "loadingOrder": "Cargando orden...",
+  "done": "HECHO",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)